src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -363,10 +363,10 @@ export class AppComponent implements OnInit, AfterViewChecked {
                     };
 
                     // Precalculate the max height and min height
-                    let maxHeight = resultSet.totalRows < self._defaultNumShowingRows
-                        ? Math.max((resultSet.totalRows + 1) * self._rowHeight, self.dataIcons.length * 30) + 10
+                    let maxHeight = resultSet.rowCount < self._defaultNumShowingRows
```

Comment:
I'm not entirely certain how to do it, but I think we could test the computed height of the children of #results when given different number of rows. We can likely address this after the current release 